gen3_tracker/meta/skeleton.py

```diff
@@ -0,0 +1,262 @@
+
+    if observation_identifier:
+        assert patient, "patient required for observation"
+        observation = Observation(status='final', code={'text': 'unknown'})
+        observation.identifier = [Identifier(value=observation_identifier, system=_get_system(observation_identifier, project_id=project_id), use='official')]
+        observation.subject = {'reference': f"Patient/{patient.id}"}
+        observation.id = create_id(observation, project_id)
+
+    if specimen_identifier:
+        specimen = Specimen()
+        specimen.identifier = [Identifier(value=specimen_identifier, system=_get_system(specimen_identifier, project_id=project_id), use='official')]
+        specimen.id = create_id(specimen, project_id)
+        assert patient, "patient required for specimen"
+        specimen.subject = {'reference': f"Patient/{patient.id}"}
+
+    if task_identifier:
+        task = Task(intent='unknown', status='completed')
+        task.identifier = [Identifier(value=task_identifier, system=_get_system(task_identifier, project_id=project_id),
+                                      use='official')]
+        task.id = create_id(task, project_id)
+
+    # create relationships
+
+    # assign subject, specimen of observation
+    if observation and specimen and not observation.specimen:
+        observation.specimen = {'reference': f"Specimen/{specimen.id}"}
+    if observation and patient and not observation.subject:
+        observation.subject = {'reference': f"Patient/{patient.id}"}
+
+    if task:
+        task.input = []
+        if specimen:
+            task.input.append(
+                TaskInput(
+                    valueReference={"reference": f"Specimen/{specimen.id}"},
+                    type={'text': 'Specimen'}
+                )
+            )
+        if patient:
+            task.input.append(
+                TaskInput(
+                    valueReference={'reference': f"Patient/{patient.id}"},
+                    type={'text': 'Patient'}
+                )
+            )
+        task.output = [
+            TaskOutput(
+                valueReference={'reference': f"DocumentReference/{document_reference.id}"},
+                type={'text': 'DocumentReference'})
+        ]
+
+    # assign document reference subject
+    if observation:
+        document_reference.subject = {'reference': f"Observation/{observation.id}"}
+    if specimen and not document_reference.subject:
+        document_reference.subject = {'reference': f"Specimen/{specimen.id}"}
+    if patient and not document_reference.subject:
+        document_reference.subject = {'reference': f"Patient/{patient.id}"}
+    if not document_reference.subject:
+        document_reference.subject = {'reference': f"ResearchStudy/{research_study.id}"}
+
+    return [_ for _ in [research_study, research_subject, patient, observation, specimen, task, document_reference] if _]
+
+
+def update_meta_files(dry_run=False, project_id=None) -> list[str]:
```

Comment:
Seems like this function should get called as a part of the commit / push process so that orphaned metadata information can be passed to the server as their corresponding data files are cleaned up. Thoughts?

Reply:
called by [`meta init`](https://github.com/ACED-IDP/gen3_util/blob/7bc05e8c2d8cfb3f01ad415c9bea068052bd96b9/gen3_tracker/meta/cli.py#L35) 

checked by [`status`] (https://github.com/ACED-IDP/gen3_util/blob/641be0f59890933eb53e93068371a8131dfe68af/gen3_tracker/git/cli.py#L268-L270)

which in turn is checked by [`push`](https://github.com/ACED-IDP/gen3_util/blob/641be0f59890933eb53e93068371a8131dfe68af/gen3_tracker/git/cli.py#L337)